core/templates/pages/library-page/search-bar/search-bar.component.css

```diff
@@ -183,9 +180,13 @@ oppia-search-bar .oppia-input-group.classroom-page-input-group {
     height: 45px;
     padding-top: 8px;
   }
-  oppia-search-bar .oppia-input-group input {
-    border-bottom-left-radius: 10px;
+  
+  .input-group > .form-control:not(:first-child){
```

Comment:
[The `not` pseudo-class also causes unusual effects, and has a very weird set of rules that need to be kept in mind when working with it.](https://developer.mozilla.org/en-US/docs/Web/CSS/:not#description) This makes things unnecessarily complicated. Could you find an alternate approach?

(Line 168 sets the border-radius to 0. Maybe you could try tweaking/removing that to see if things work?

Or maybe you could increase the specifity of this ^ rule in some other manner?)